File: 3450-FindTheChildWhoHasTheBallAfterKSeconds/3450-FindTheChildWhoHasTheBallAfterKSeconds.py
# Last updated: 7/31/2025, 5:12:08 PM
class Solution:
    def numberOfChild(self, n: int, k: int) -> int:
        k=k%(2*n-2)
        if k>=n:
            return 2*n-2-k
        return k
        

File: 3450-FindTheChildWhoHasTheBallAfterKSeconds/3450-FindTheChildWhoHasTheBallAfterKSeconds.java
// Last updated: 7/31/2025, 5:11:47 PM
class Solution {
    public int numberOfChild(int n, int k) {
        k=k%(2*n-2);
        if(k<n){
            return k;
        }
        return 2*n-2-k;
        
    }
}